compiler/src/dmd/expressionsem.d

```diff
@@ -7087,7 +7087,7 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
             sc2.tinst = null;
             sc2.minst = null;
             sc2.flags |= SCOPE.fullinst;
-            Type t = e.targ.trySemantic(e.loc, sc2);
+            Type t = dmd.typesem.trySemantic(e.targ, e.loc, sc2);
```

Comment:
is it really necessary to fully qualify this call?

Reply:
Yes, because expressionsem also defines trySemantic on an Expression and the compiler prefers the local declaration as opposed to the imported one. Also, it cannot be included in the same overload set via alias because then you would have problems with the trySemantic invokations on types from typesem.d . Simply put, the `trySemantic` overload set should be defined in a module that is imported by both expressionsem and typesem, but since such a module is not easy to find, I opted to keeping each trySemantic local to the semantic routines.

Reply:
OK